CHANGELOG.rst

```diff
@@ -1,6 +1,15 @@
 CHANGELOG
 =========
 
+1.0.3 (2015-07-22)
+------------------
+
+* Numerous fixes for search indexing support including a switch to disable the
```

Comment:
"support including a switch to disable the auto-generation of search "pre-indexing"." - was done in 1.0.2